pkg/providers/cloudstack/testdata/expected_results_main_cp.yaml

```diff
@@ -389,18 +389,23 @@ metadata:
 spec:
   replicas: 3
   etcdadmConfigSpec:
-    etcdadmBuiltin: true
+    etcdadmBuiltin: false
     format: cloud-config
     cloudInitConfig:
       version: 3.4.16
       installDir: "/usr/bin"
+    etcdadmInstallCommands:
+      - echo this line exists so that etcdadmInstallCommands is not empty
+      - echo etcdadmInstallCommands can be removed once etcdadm bootstrap and controller fix the bug
+      - echo that preEtcdadmCommands not run unless etcdadmBuiltin is false
     preEtcdadmCommands:
     - swapoff -a
     - hostname "{{ ds.meta_data.local_hostname }}"
     - echo "::1         ipv6-localhost ipv6-loopback" >/etc/hosts
     - echo "127.0.0.1   localhost" >>/etc/hosts
     - echo "127.0.0.1   {{ ds.meta_data.local_hostname }}" >>/etc/hosts
     - echo "{{ ds.meta_data.local_hostname }}" >/etc/hostname
+    - echo "type=83" | sfdisk /dev/vdb && mkfs -t ext4 /dev/vdb1 && mkdir -p /data-small && echo /dev/vdb1 /data-small ext4 defaults 0 2 >> /etc/fstab && mount /data-small
```

Comment:
could you add some comments here explaining what's happening? Maybe split across multiple lines?

Reply:
This is the temp solution before we add disksetup and mount support in etcdadmconfig. 
I managed to split them into multiple lines so that is easier to read, but could not insert comments in it. 

Anyway, it should be much easier to understand even without comments.